surveys/views.py

```diff
@@ -67,18 +67,31 @@ def post(self, request, **kwargs):
         context['survey'].published = True
         context['survey'].save()
 
-        return redirect('surveys-list')
+        return redirect('surveys-list-run')
 
 
-class SurveyList(ListView):
+class SurveyListEdit(ListView):
     model = SurveyFormEntry
     template_name = "survey_list.html"
     context_object_name = 'surveys'
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context['published'] = self.request.GET.get('published', 'f')
-        context['surveys'] = context['surveys'].filter(published=context['published'])
+        context['surveys'] = context['surveys'].filter(published=False)
+        context['published'] = False
```

Comment:
Note that we could also accomplish this filtering in the [`ListView.get_queryset()`](https://docs.djangoproject.com/en/dev/ref/class-based-views/mixins-multiple-object/#django.views.generic.list.MultipleObjectMixin.get_queryset) method. Both approaches work fine, but if at some point this method starts to get unweildy, you could factor out the portions that alter the `surveys` object into `get_queryset()`.